### PR TITLE
Release @okta/odyssey 0.8.0

### PR DESCRIPTION
**Pull request by jeffbelser-okta**

### CODE REVIEW

File: packages/odyssey/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
...
+- Move stylelint dependencies into odyssey-stylelint package
+- fix: invalid checkbox disabled state (#933)
+- Legal updates for OSS
+- "Alert" components group (#913)
+- respect prefers-reduced-motion (#928)
+- Refactor workspace scripts to be more granular
+- Updates Odyssey Prism theme (#919)
+- add contextual styles for full width buttons (#911)
+- Add full-width button variant (#910)
+- Foundations: Update Type Scale (#903)
```
Comment (edburyenegren-okta) [inline packages/odyssey/CHANGELOG.md:34]:
```suggestion
## [0.8.0] - 2021-07-27

### Added

- Added additional Button sizes and widths (#985, #910, #991)
- Added "Alert" components group: Banner, Infobox, and updated Toast (#913)
- Added support for `prefers-reduced-motion` (#928)
- Added RTL linting

### Changed

- Type Scale 4.0 (#903, #931)
  - Arabic-safe typography (#993)
  - Base font-size is now 16px
  - Base font-family is now Public Sans
  - Line-height updates to improve readability
- [Breaking Change] Refactor: RTL Support (#984, #968)
  - Updates all classes to use dir-agnostic names ("left" to "start")
  - Updates all relevant properties to use logical CSS
- [Breaking Change] prefer `:invalid` to data attr (#957)
- Updated focus state for dismiss button
- Allow stylelint to be invoked from workspace root
- Move stylelint dependencies into odyssey-stylelint package
- Legal updates for OSS
- Refactor workspace scripts to be more granular
- Updates Odyssey Prism theme (#919)

### Removed

- Removes refs to Whyte, prefers link to import (#960)

### Fixed

- Docs: Minor cleanup items from the Typography audit (#949)
- Radio button disabled no longer reacts to hover (#948)
- Checkbox ui invalid+disabled+unchecked state combinations now display correctly (#933, #943)
- Fix stylelint errors re: OSS headers
```

Comment (edburyenegren-okta) [reply packages/odyssey/CHANGELOG.md:34]: Proposed change above to make our changelog match prior formats until we have a new standard.

**Comment by jeffbelser-okta** [comment]
:soon:

File: packages/odyssey/CHANGELOG.md (hunk 1 above)
Comment (jeffbelser-okta) [reply packages/odyssey/CHANGELOG.md:34]: Is this a markdown refactor? Nice Thanks for doing that. Yes this needs some more love next time we do this for real. We'll include it as part of our future release work cc @edburyenegren-okta

**Pull request closed without merging**